lib/steps/rollup.ts

```diff
@@ -27,151 +29,26 @@ export const rollup = (opts: RollupOptions) => {
-    'rxjs/add/operator/publishReplay':  'Rx.Observable.prototype',
-    'rxjs/add/operator/race':           'Rx.Observable.prototype',
-    'rxjs/add/operator/reduce':         'Rx.Observable.prototype',
-    'rxjs/add/operator/repeat':         'Rx.Observable.prototype',
-    'rxjs/add/operator/repeatWhen':     'Rx.Observable.prototype',
-    'rxjs/add/operator/retry':          'Rx.Observable.prototype',
-    'rxjs/add/operator/retryWhen':      'Rx.Observable.prototype',
-    'rxjs/add/operator/sample':         'Rx.Observable.prototype',
-    'rxjs/add/operator/sampleTime':     'Rx.Observable.prototype',
-    'rxjs/add/operator/scan':           'Rx.Observable.prototype',
-    'rxjs/add/operator/sequenceEqual':  'Rx.Observable.prototype',
-    'rxjs/add/operator/share':          'Rx.Observable.prototype',
-    'rxjs/add/operator/single':         'Rx.Observable.prototype',
-    'rxjs/add/operator/skip':           'Rx.Observable.prototype',
-    'rxjs/add/operator/skipUntil':      'Rx.Observable.prototype',
-    'rxjs/add/operator/skipWhile':      'Rx.Observable.prototype',
-    'rxjs/add/operator/startWith':      'Rx.Observable.prototype',
-    'rxjs/add/operator/subscribeOn':    'Rx.Observable.prototype',
-    'rxjs/add/operator/switch':         'Rx.Observable.prototype',
-    'rxjs/add/operator/switchMap':      'Rx.Observable.prototype',
-    'rxjs/add/operator/switchMapTo':    'Rx.Observable.prototype',
-    'rxjs/add/operator/take':           'Rx.Observable.prototype',
-    'rxjs/add/operator/takeLast':       'Rx.Observable.prototype',
-    'rxjs/add/operator/takeUntil':      'Rx.Observable.prototype',
-    'rxjs/add/operator/takeWhile':      'Rx.Observable.prototype',
-    'rxjs/add/operator/throttle':       'Rx.Observable.prototype',
-    'rxjs/add/operator/throttleTime':   'Rx.Observable.prototype',
-    'rxjs/add/operator/timeInterval':   'Rx.Observable.prototype',
-    'rxjs/add/operator/timeout':        'Rx.Observable.prototype',
-    'rxjs/add/operator/timeoutWith':    'Rx.Observable.prototype',
-    'rxjs/add/operator/timestamp':      'Rx.Observable.prototype',
-    'rxjs/add/operator/toArray':        'Rx.Observable.prototype',
-    'rxjs/add/operator/toPromise':      'Rx.Observable.prototype',
-    'rxjs/add/operator/window':         'Rx.Observable.prototype',
-    'rxjs/add/operator/windowCount':    'Rx.Observable.prototype',
-    'rxjs/add/operator/windowToggle':   'Rx.Observable.prototype',
-    'rxjs/add/operator/windowWhen':     'Rx.Observable.prototype',
-    'rxjs/add/operator/withLatestFrom': 'Rx.Observable.prototype',
-    'rxjs/add/operator/zipAll':         'Rx.Observable.prototype',
-    'rxjs/add/operator/zipProto':       'Rx.Observable.prototype',
+    // external symbols passed from the user's ng-package.json
     ...opts.externals,
   };
 
+  const ROLLUP_COMMONJS_INCLUDE = [
+    // RxJS dependencies
+    'node_modules/rxjs/**',
+    // commonjs symbols passed from the user's ng-package.json
+    ...opts.commonjs
+  ];
```

Comment:
`rollup-plugin-commonjs` does have an issue w/ monorepos.

```bash
$ cd packages/my-package
$ ../../node_modules/.bin/ng-packagr
```

Now, rollup will resolve the dependency to `../../node_modules/rxjs` but the value in `ROLLUP_COMMONJS_INCLUDE ` does not match.

In result, together w/ removing the default `externals` breaks existing builds.